src/OrchardCore.Modules/OrchardCore.ContentPreview/Views/ContentPreview.Button.cshtml

```diff
@@ -1,20 +1,18 @@
-@using Microsoft.AspNetCore.Authorization
 @using OrchardCore.ContentManagement
-@using OrchardCore.ContentPreview
 @inject IAuthorizationService AuthorizationService
 
 @{
     ContentItem contentItem = Model.ContentItem;
     var previewId = Guid.NewGuid();
-    var hasPreviewPermission = await AuthorizationService.AuthorizeAsync(User, Permissions.ContentPreview, contentItem);
+    var hasPreviewPermission = await AuthorizationService.AuthorizeAsync(User, OrchardCore.Contents.CommonPermissions.PreviewContent, contentItem);
```

Comment:
The comparison done here is against `Permission.Name`, but changing this makes the permission Permissions.ContentPreview uselessish.

Reply:
Permissions.ContentPreview and CommonPermissions.PreviewContent should be the same?

Reply:
Maybe not I'll take a look

Reply:
Ok they are not the same. I'm looking if I did a mistake. If I understand Permissions.ContentPreview is a global permission. Then for OrchardCore.Contents module we have CommonPermissions.PreviewContent which is also a Global permission for all content types. So, basically it tells that we can disable the Preview globally in 2 places.

Permissions.ContentPreview is used in only 2 places : 
Here and in the PreviewController. So I believe that it should be replaced by OrchardCore.Contents.CommonPermissions.PreviewContent in both places.

Reply:
Ok removed the OrchardCore.ContentPreview Permission.cs file.